clients/admin-ui/src/features/common/privacy-notice-regions.ts

```diff
@@ -0,0 +1,120 @@
+import { PrivacyNoticeRegion } from "~/types/api";
+
+/**
+ * A mapping of privacy notice regions to human readable names.
+ *
+ * The typing is important here so that our build will fail if we are missing a region.
+ */
+export const PRIVACY_NOTICE_REGION_RECORD: Record<PrivacyNoticeRegion, string> =
```

Comment:
please review this carefully—would be embarrassing if I put a place that was i.e. EU but isn't actually!

Reply:
I checked carefully! Everything is correct. We are not listing US territories but that should be a separate conversation with product and PR.